funcx_web_service/models/search.py

```diff
@@ -16,15 +17,22 @@
 DEFAULT_SEARCH_LIMIT = 10
 
 
+def _sanitize_tokens(token_data: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Is it worth considering what happens if other tokens are added to this dict at some point? Since you copy the entire dict and then sanitize only the ones we know about there is the risk that other tokens could be returned unscathed.

Reply:
At the moment, the only other kind of token returned by auth would be an `id_token`, and while those will never be returned for the search scope and also are less of a concern from a security point of view, it might be worth filtering out as well.  

And to Ben's point, it might be better to be explicit about what we keep from the token response rather then implicitly keeping everything not filtered.  E.g. something like:
```
def _sanitize_tokens(token_data: Dict[str, Any]) -> Dict[str, Any]:                                                                                                                                                                                                                                                        
     return {                                                                                                                                                                                                                                                                                                               
         'scope': token_data.get('scope'),                                                                                                                                                                                                                                                                                  
         'access_token': token_data.get('access_token', "")[-5:],                                                                                                                                                                                                                                                           
         'refresh_token': token_data.get('refresh_token', "")[-5:],                                                                                                                                                                                                                                                         
     }
```

Reply:
Those are good points -- I went the route of returning a dict that only pulls the fields we want for logging (which feels like a nice layer of further sanitizing). If we end up needing to pull more fields from the token it should be fairly easy to add them.